gramps/gui/viewmanager.py

```diff
@@ -580,10 +580,47 @@ def no_del_event(self, *obj):
         hits 'x' multiple times. """
         return True
 
+    def checkdisk(self, dbfolder): # TODO check this function
```

Comment:
This belongs in [`gramps/gen/utils/file.py`](https://github.com/gramps-project/gramps/blob/master/gramps/gen/utils/file.py).

Reply:
Yes, in new PR this function move to file.py.